src/components/UserProfile/index.tsx

```diff
@@ -0,0 +1,35 @@
+import UserSignalTemperature from './UserSignalTemperature';
+
+interface UserProfileProps {
+  nickname: string;
+  profileImageUrl: string;
+  ageGroup: string;
+  value: number;
+}
+
+const UserProfile = ({
+  nickname,
+  profileImageUrl,
+  ageGroup,
+  value,
+}: UserProfileProps) => {
+  const DEFAULT_PROFILE_IMAGE_URL = 'https://picsum.photos/200/200';
+
+  return (
+    <div className='flex'>
+      <img
+        src={profileImageUrl || DEFAULT_PROFILE_IMAGE_URL}
+        className='h-10 w-10 rounded-full object-cover'
+        style={{ marginRight: '0.5rem' }}
```

Comment:
`className`으로 작성해도 되는 스타일 같아요!

```suggestion
        className='h-10 w-10 rounded-full object-cover mr-2'
```

Reply:
적용이 안돼서 스타일로 뺐었는데 다시 해보니 적용되네요ㅠㅠ감사합니다!!